echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/monitor/TriggerEventListener.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.echo.pipelinetriggers.monitor;
+
+import com.netflix.spectator.api.Registry;
+import com.netflix.spinnaker.echo.events.EchoEventListener;
+import com.netflix.spinnaker.echo.model.Event;
+import com.netflix.spinnaker.echo.pipelinetriggers.PipelineCache;
+import com.netflix.spinnaker.echo.pipelinetriggers.eventhandlers.TriggerEventHandler;
+import com.netflix.spinnaker.echo.pipelinetriggers.orca.PipelineInitiator;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+/**
+ * Triggers pipelines on _Orca_ when a trigger-enabled build completes successfully.
+ */
+@Slf4j
+@Component
+public class TriggerEventListener implements EchoEventListener {
+  private final List<TriggerMonitor> triggerMonitors;
+
+  public TriggerEventListener(@NonNull PipelineCache pipelineCache,
+    @NonNull PipelineInitiator pipelineInitiator,
+    @NonNull Registry registry,
+    @NonNull List<TriggerEventHandler<?>> eventHandlers
+  ) {
```

Comment:
The general rule of thumb for these parens is: "the closing paren is preceded by a newline i.f.f. the opening paren is followed by a newline".